### DISCUSSION TITLE
[Project-BETA] Batch edit project items

### DISCUSSION THREAD

**Discussion by Smail** [community/community#7019] [Projects and Issues]
Right now, if you create multiple items in a project, you have to individually set their column values, e.g. you have to set all new items to "todo" in the status column one by one. It would be nice if you could select multiple items (with shift click, for example) and edit them all at once. A default value for a column we be great as well.

Edit: this is a duplicate of #6528

**Comment by Aarbel** [comment]
Would be really useful to:
- bulk drag and drop issues
- apply labels / columns to bulk tickets 

Right now only archive / delete is available 

<img width="477" alt="image" src="https://user-images.githubusercontent.com/25119847/232712063-2e758405-6ec8-4941-b132-f53cea3034ad.png">

**Reply by spookyuser** [reply]
And this also doesn't work in table view
